controller/views/index.html.erb

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
```

Comment:
wow! there's a lot of work here - I'm tempted to clone your repo and see what it looks like :)